src/config.js

```diff
@@ -1,10 +1,10 @@
 // @flow
 
-const HOST = 'https://brewskey.com';
-// const HOST = 'http://10.0.2.2:2484';
+const HOST = 'https://brewskey.com/';
```

Comment:
I'm pretty sure that I removed the `/` because signalr had multiple slashes in the URL.

You will need to fix these places with this change:
https://github.com/Brewskey/Brewskey.App/commit/54007542811b18ad3f2240d1a6f674d4a182395b

Reply:
yeah, you 're right, I'm going to revert slashes back...signalr host is hardcoded atm in brewskey.js-api though, because it creates its connection in constructor..I'll fix that in js-api side later.